### PR TITLE
[DRAFT] java-quarkus devfile

### PR DESCRIPTION
**Pull request by ibuziuk**
### What does this PR do?:
_Summarize the changes. Are any stacks or samples added or updated?_

### Which issue(s) this PR fixes:
_Link to github issue(s)_

### PR acceptance criteria:

- [ ] Contributing guide
_Have you read the [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) and followed its instructions?_
- [ ] Test automation
_Does this repository's tests pass with your changes?_
- [ ] Documentation
_Does any documentation need to be updated with your changes?_

### How to test changes / Special notes to the reviewer:

### CODE REVIEW

File: stacks/java-quarkus/devfile.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ components:
   - name: tools
     container:
       image: registry.access.redhat.com/ubi8/openjdk-11
-      memoryLimit: 512Mi ## default app nowhere needs this but leaving room for expansion.
+      memoryLimit: 10Gi ## 'init-compile' command needs at very least 10Gi of RAM.
```
[the remark this answers is not in the document]
Comment (johnmcollier) [reply stacks/java-quarkus/devfile.yaml:23]: Yeah... this'll likely mean we can't run the java-quarkus devfile in our tests on Github, which limit the total memory to 7GB

[the remark this answers is not in the document]
Comment (johnmcollier) [reply stacks/java-quarkus/devfile.yaml:23]: CC @maxandersen

[the remark this answers is not in the document]
Comment (maxandersen) [reply stacks/java-quarkus/devfile.yaml:23]: Why do we need that kind of jump?

[the remark this answers is not in the document]
Comment (maxandersen) [reply stacks/java-quarkus/devfile.yaml:23]: Like - no way any of this requires 10GB from quarkus pov. Native compile is sub 5 GB and that is not what this devfile is trying to support. 

What am I missing here ? What part requires 10GB ?

**Comment by maxandersen** [comment]
can we close this? no updates or info on why memory requirements so high?

**Pull request closed without merging**